pages/models.py

```diff
@@ -99,8 +99,8 @@ class Page(MPTTModel):
             blank=True)
 
     freeze_date = models.DateTimeField(_('freeze date'),
-            null=True, blank=True, help_text=_('''Don't publish any content
-            after this date.'''))
+            null=True, blank=True, help_text=_("Don't publish any content "
+            "after this date."))
```

Comment:
Is there a reason for changing the string here? Is it a best practice?

Reply:
I just wanted to escape newline and tabs in string, so it is not showing in localization editor.